### ISSUE TITLE
Allow upload of fallback video file types for background videos

### ISSUE THREAD

**Issue by hcavanaugh** [elementor/elementor#8941]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

Beaver builder has a setup where you can upload a link to a .webm video file, .mp4 file, and a fallback image. Elementor currently only has one video link upload and fallback image. .webm is a nice file format, but isn't supported by Safari. I want to be able to have fallback video files for these users.

**Comment by gitgudcameron** [comment]
I can't believe this issue was posted almost a year ago and still no response! It should be standand practice to be able to use webm and MP4 formats, especially for background videos.

**Comment by addMitt** [comment]
Came here to suggest this same thing. Webm's are great and can be much smaller than mp4's, and for anyone designing their own video backgrounds that aren't hosted on YouTube, this feature is an absolute must.

Background videos and the Video widgets self-hosted video option should both probably have this option.

**Comment by AndersJpg** [comment]
could really use this by now :/

**Comment by jccbbb** [comment]
This should be something that isn´t all too hard to implement for the Elementor developers and also very important for a good UX for those who (unfortunately) use Safari and older browsers.

**Comment by markbosky** [comment]
+1
.webm is superior to .mp4, but not all browsers support it.

**Comment by sociobright** [comment]
Same, It's been 4 years, and the need for webm and fallback videos is higher than ever, yet the Elementor team chooses to ingore this,.

**Comment by andreaskasper** [comment]
Would love to have this feature. Including multiple bitrates/sizes depending on the screen-size.

**Comment by kashann** [comment]
Sad to see that this development was not started yet, taking into account the performance improvement that can be achieved with it...
A lot of end users would benefit from this.

**Comment by tikitock** [comment]
YES!  makes webm useless since no easy way to create this with fallback

**Comment by hschk** [comment]
Is there any progression on that? We are in 2024 now and still not able to set a fallback Video?
Same goes for the normal video widget btw...
Please! This is really simple to implement and would be a huge improvement.

**Comment by wecode-swiss-bruno** [comment]
Any update please?

**Comment by Joker2099** [comment]
This is still needed very much.
